### ISSUE TITLE
CALL METHOD, missing method not reported

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#2186]
```abap
CLASS lcl_bar DEFINITION.
ENDCLASS.
CLASS lcl_bar IMPLEMENTATION.
ENDCLASS.

DATA out TYPE REF TO lcl_bar.
CALL METHOD out->something( ).
```

![image](https://user-images.githubusercontent.com/5888506/136333707-cb6e6505-51a3-47b6-8556-6d6ff6788ee3.png)

but

![image](https://user-images.githubusercontent.com/5888506/136333733-7dcf0c62-8a5a-4301-9ab2-71a6fd02dd8c.png)
